yt_dlp/extractor/gettr.py

```diff
@@ -0,0 +1,124 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    dict_get,
+    float_or_none,
+    int_or_none,
+    remove_end,
+    str_or_none,
+    try_get,
+    url_or_none,
+    urljoin,
+)
+
+
+class GettrIE(InfoExtractor):
+    _VALID_URL = r'https?://(www\.)?gettr\.com/post/(?P<id>[a-z0-9]+)'
+    _MEDIA_BASE_URL = 'https://media.gettr.com/'
+
+    _TESTS = [{
+        'url': 'https://www.gettr.com/post/pcf6uv838f',
+        'info_dict': {
+            'id': 'pcf6uv838f',
+            'title': 'md5:0f10d7d504f92b1a59fd52a0d62cbc9c',
+            'description': 'md5:be0577f1e4caadc06de4a002da2bf287',
+            'ext': 'mp4',
+            'uploader': 'EpochTV',
+            'uploader_id': 'epochtv',
+            'thumbnail': r're:^https?://.+/out\.jpg',
+            'timestamp': 1632782451058,
+            'duration': 58.5585,
+        }
+    }, {
+        'url': 'https://gettr.com/post/p4iahp',
+        'info_dict': {
+            'id': 'p4iahp',
+            'title': 'md5:b03c07883db6fbc1aab88877a6c3b149',
+            'description': 'md5:741b7419d991c403196ed2ea7749a39d',
+            'ext': 'mp4',
+            'uploader': 'Neues Forum Freiheit',
+            'uploader_id': 'nf_freiheit',
+            'thumbnail': r're:^https?://.+/out\.jpg',
+            'timestamp': 1626594455017,
+            'duration': 23,
+        }
+    }]
+
+    def _real_extract(self, url):
+        post_id = self._match_id(url)
+        webpage = self._download_webpage(url, post_id)
+
+        api_data = self._download_json(
+            'https://api.gettr.com/u/post/%s?incl="poststats|userinfo"' % post_id, post_id)
+
+        post_data = try_get(api_data, lambda x: x['result']['data'])
+        user_data = try_get(api_data, lambda x: x['result']['aux']['uinf'][post_data.get('uid')]) or {}
```

Comment:
```suggestion
        user_data = try_get(api_data, lambda x: x['result']['aux']['uinf'][post_data['uid']]) or {}
```
no point using `.get` inside `try_get`